<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmIpcAddDialog</class>
 <widget class="QDialog" name="frmIpcAddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>操作员登录</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetTitle" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labIco">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labTitle">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>添加其他设备</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnMenu_Close">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetMain" native="true">
     <property name="styleSheet">
      <string notr="true">#gboxSoftKey .QPushButton{
border-width:0px;
border-radius:0px;
padding:2px;
}

#gboxSoftKey .QPushButton:hover{
border:1px solid #FF0000;
}

#gboxSoftKey{
border-width:0px;
border-radius:0px;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0" colspan="4">
          <widget class="QTabWidget" name="tabWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tabAddIpc">
            <attribute name="title">
             <string>批量添加摄像机</string>
            </attribute>
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>10</y>
               <width>461</width>
               <height>261</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <property name="horizontalSpacing">
               <number>10</number>
              </property>
              <property name="verticalSpacing">
               <number>15</number>
              </property>
              <item row="3" column="0">
               <widget class="QLabel" name="labelUsrName">
                <property name="text">
                 <string>用户名</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1" colspan="3">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>不同厂家端口不一样 一般取值554、443、80、8080</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="labelIpcBrand">
                <property name="text">
                 <string>摄像机品牌</string>
                </property>
               </widget>
              </item>
              <item row="6" column="2">
               <widget class="QLabel" name="labelSubResolution">
                <property name="text">
                 <string>子码流</string>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QLineEdit" name="lineEditEndIP"/>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="labelPort">
                <property name="text">
                 <string>端口</string>
                </property>
               </widget>
              </item>
              <item row="6" column="3">
               <widget class="QComboBox" name="cbBoxSubResolution"/>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="labelMainResolution">
                <property name="text">
                 <string>主码流</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1" colspan="3">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>例如： 192.168.1.1 到 192.168.1.200</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="cbBoxIpcBrand"/>
              </item>
              <item row="1" column="2">
               <widget class="QLabel" name="labelEndIP">
                <property name="text">
                 <string>结束ip</string>
                </property>
               </widget>
              </item>
              <item row="3" column="3">
               <widget class="QLineEdit" name="lineEditPwd"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="labelStartIp">
                <property name="text">
                 <string>起始IP</string>
                </property>
               </widget>
              </item>
              <item row="6" column="1">
               <widget class="QComboBox" name="cbBoxMainResolution"/>
              </item>
              <item row="4" column="1">
               <widget class="QLineEdit" name="lineEditPort"/>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="lineEditUsrName"/>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="lineEditStartIP"/>
              </item>
              <item row="3" column="2">
               <widget class="QLabel" name="labelPwd">
                <property name="text">
                 <string>密码</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
           <widget class="QWidget" name="tab_2">
            <attribute name="title">
             <string>HDMI添加</string>
            </attribute>
           </widget>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="btnCancle">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>取消</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="btnAddIpc">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>开始添加</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
